src/node.cpp

```diff
@@ -1020,6 +1054,19 @@ bool Node::setparent(Node* p)
     if (parent)
     {
         child_it = parent->children.insert(parent->children.end(), this);
+
+#ifdef ENABLE_SYNC
+        if (parent->type == FILENODE)
+        {
+            const auto syncPath = client->unsyncables->syncPath(nodehandle);
```

Comment:
could we test child->isSyncable here instead?

Reply:
done.